.experimenter.yaml

```diff
@@ -1,12 +1,4 @@
 ---
-default-browser-message:
-  description: A small feature allowing experiments on the placement of a default browser message.
-  hasExposure: true
-  exposureDescription: ""
-  variables:
-    message-location:
-      type: string
-      description: Where is the message to be put.
```

Comment:
@jhugman I found we are still using this for the default browser menu item on the toolbar.

https://github.com/mozilla-mobile/fenix/blob/7d37c2bcbd76628bf2e33fbf8a246085e2352abf/app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt#L453-L468

Reply:
This relates to the other bug: https://mozilla-hub.atlassian.net/browse/FNXV2-20703 . I think I'd ni your product person and ask, but I would expect you should remove this at minimum, or convert it to another message surface later on.